src/utils/Disposable.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Based off of the Node worker
+ * https://github.com/Azure/azure-functions-nodejs-worker/blob/bf28d9c5ad4ed22c5e42c082471d16108abee140/src/Disposable.ts
+ */
+export class Disposable {
+    static from(...inDisposables: { dispose(): any }[]): Disposable {
```

Comment:
I see in the link above that we have the same call signature, but I noticed in [this file](https://github.com/Azure/azure-functions-nodejs-library/blob/c3aa2c4c2cff28317e00ff23460758b47c7c47f5/types-core/index.d.ts#L159) that it is slightly different. Should we align all three? Or is there a reason the example in `types-core` is slightly different?

Reply:
It's actually based on VS Code, not from the worker. I'll update the comment and get the latest VS Code source. They have the same inconsistency (source [here](https://github.com/microsoft/vscode/blob/7bed4ce3e9f5059b5fc638c348f064edabcce5d2/src/vs/workbench/api/common/extHostTypes.ts#L65) and type [here](https://github.com/microsoft/vscode/blob/main/src/vscode-dts/vscode.d.ts#L1675)). I don't think the inconsistency between the source and the type matters in a functional sense, so I'd rather stay consistent between us and VS Code